.github/workflows/commit.yaml

```diff
@@ -82,3 +82,37 @@ jobs:
         env:
           CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
         run: bash <(curl -s https://codecov.io/bash)
+
+  test-windows:
+    name: "Run unit tests (windows-latest)"
+    runs-on: windows-latest
+    timeout-minutes: 90  # instead of 360 by default
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v2
+
+      - name: "Install Go"
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+
+      - name: "Cache dependencies"
+        uses: actions/cache@v2
+        with:
+          # This combines unrelated caches because actions/cache@v2 doesn't support multiple
+          # instances, rather a combined path. https://github.com/actions/cache/issues/16
+          path: |  # ~\.func-e\versions is cached so that we only re-download once: for TestFuncEInstall
+            ~\.func-e\versions
+            ~\go\pkg\mod
+          key: test-windows-latest-${{ env.GO_VERSION }}-go-${{ hashFiles('internal/version/last_known_envoy.txt', 'go.sum') }}
```

Comment:
the actions/cache example for windows uses unix slashes in the hashFiles thing, so did the same thing